platform/broadcom/platform-modules-arista.mk

```diff
@@ -4,6 +4,7 @@ ARISTA_PLATFORM_MODULE_VERSION = 1.0
 
 export ARISTA_PLATFORM_MODULE_VERSION
 
+$(DOCKER_PLATFORM_MONITOR)_aboot_RUN_OPT += -v /usr/lib/python2.7/dist-packages/arista:/usr/lib/python2.7/dist-packages/arista:ro
```

Comment:
can you move this to docker-platform-monitor.mk? It is a docker platform monitor run option, should be in the file for future maintainance

Reply:
done.
It was originally put in platform-modules-arista.mk since the extra option only make sense if this package is installed.